src/SFA.DAS.EmployerIncentives.Api/Authentication/AuthenticationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerIncentives.Configuration;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authentication
+{
+    public static class AuthenticationExtensions
```

Comment:
Does this class, or similar, not exist in one of the projects already in the solution?  If it does, can we reuse that instead?

Reply:
Thjere was no shared classes in the code when I started. Will discuss with Dan, See what they are doing

Reply:
Will be in the shared project eventually